src/display/api_compatibility.js

```diff
@@ -35,16 +32,6 @@ if (typeof PDFJSDev === 'undefined' || PDFJSDev.test('GENERIC')) {
     }
   })();
 
-  // Support: Safari 6.0+, iOS
-  (function checkRangeRequests() {
-    // Safari has issues with cached range requests, see issue #3260.
-    // Last tested with version 6.0.4.
-    if (isSafari || isIOS) {
-      compatibilityParams.disableRange = true;
-      compatibilityParams.disableStream = true;
```

Comment:
With these now being removed, all occurrences of `apiCompatibilityParams.disableRange`/`apiCompatibilityParams.disableStream` throughout the code-base will always be `undefined` and ought to be removed as well.

See also https://github.com/mozilla/pdf.js/wiki/Squashing-Commits